tests/flow/test_procedures.py

```diff
@@ -371,6 +371,24 @@ def test12_procedure_reordered_yields(self):
                            ["READ",  "db.labels"],
                            ["READ",  "db.propertyKeys"],
                            ["READ",  "db.relationshipTypes"],
+                           ["READ",  "dbms.functions"],
                            ["READ",  "dbms.procedures"]]
         self.env.assertEquals(actual_resultset, expected_result)
 
+    def test13_functions(self):
```

Comment:
Combine this test function with the following one.
Check for at least one more function in addition to `abs`